src/components/organisms/about/about.scss

```
@@ -0,0 +1,36 @@
+@use "../../variables" as *;
+@use "../../molecules/message/message.scss" as *;
+
+.about-container {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  @media (max-width: $breakpoint-lg) {
+    padding-top: 40px;
+    padding-bottom: 40px;
```

paddingは１行でまとめて書く方がよく見られます。
```suggestion
  padding: 100px 0;
  @media (max-width: $breakpoint-lg) {
    padding: 40px 0;
```